### PR TITLE
Update: VMware.WorkstationPro version 16.2.3.21887

### PR DESCRIPTION
**Pull request by virtualex-itv**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54834)

### CODE REVIEW

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.1 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPro
+PackageVersion: 16.2.3.21887
```
Comment (ItzLevvie) [inline manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml:5]:
```suggestion
PackageVersion: 16.2.3
```
This must match Control Panel.

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.1 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPro
+PackageVersion: 16.2.3.21887
```
Comment (ItzLevvie) [inline manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 16.2.3
```
This must match Control Panel.

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.1 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPro
+PackageVersion: 16.2.3.21887
```
Comment (ItzLevvie) [inline manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.yaml:5]:
```suggestion
PackageVersion: 16.2.3
```
This must match Control Panel.

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml:5]: You will also need to update the version folder from `16.2.3.21887` to `16.2.3`

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.locale.en-US.yaml:5]: You will also need to update the version folder from `16.2.3.21887` to `16.2.3`

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.yaml:5]: You will also need to update the version folder from `16.2.3.21887` to `16.2.3`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  - PackageIdentifier: Microsoft.VC++2015-2019Redist-x86
+AppsAndFeaturesEntries:
+- Publisher: VMware, Inc.
+  DisplayName: VMware Workstation
+  InstallerType: msi
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download3.vmware.com/software/WKST-1623-WIN-New/VMware-workstation-full-16.2.3-19376536.exe
+  InstallerSha256: 117AD0B17335032D1B03BA44B03DA2128D29C65925EA79EFD2E03786B0129E94
+  ProductCode: '{13421077-BD32-4B28-96EA-354654CB7E0C}'
```
Comment (ItzLevvie) [inline manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml:32]:
```suggestion
  ProductCode: '{9D6C87F1-2E07-4B8D-99D8-D00248F62D55}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by virtualex-itv** [comment]
Not sure why this fails validation. URL is correct.

**Comment by virtualex-itv** [comment]
can someone please look at this and advise why it's failing.  this should be working.  thanks.

**Comment by denelon** [comment]
Our "on-call" will take a look at this on Monday.

**Comment by virtualex-itv** [comment]
@denelon - any updates?

**Comment by yao-msft** [comment]
The internal installer scan service reported there may be potentially unwanted behavior detected in the installer. We have reached out to the team for investigation.

**Comment by virtualex-itv** [comment]
someone needs to take a look at this manually to determine what the issue is.  The URL used in the package is indeed valid and is the direct link from VMware.  This is the same URL also used in the chocolatey package as well and there are no issues with that package.  Please advise.

**Comment by OfficialEsco** [comment]
The issue is not the link, its that some wild AV Vendor have flagged it as PUP..

**Comment by ItzLevvie** [comment]
@wingetbot run

If it errors out you will need to comment again so it doesn't automatically close the PR in 14 days.

This issue happens because there's an (or multiple) anti-virus software that has marked VMware's installers as a Potentially Unwanted Program (PUP) or Potentially Unwanted Application (PUA).

It's been an issue for previous VMware installers, and is a false positive. You can check out https://github.com/microsoft/winget-pkgs/pull/34577#event-5598809516 so you'll have to play the waiting game here.

**Comment by ItzLevvie** [comment]
> this may need manual approval.

This won't be possible as the policies require apps to pass anti-virus tests.

**Comment by virtualex-itv** [comment]
please manually approve, there is nothing wrong with the package.  it has been tested as per documentation

**Comment by denelon** [comment]
We are still getting a blocking detection in our automated scan.

**Comment by virtualex-itv** [comment]
> We are still getting a blocking detection in our automated scan.

re-running the same failing automated scans is not going to correct/resolve the issue.  what can be done to address this manually?  I manage the same package on chocolatey and have zero issues with the URL or any blocking as the file is being pulled directing from the manufacturer site.

**Comment by denelon** [comment]
The detection needs to be changed to a false positive, or the binary needs to have the detected code removed.

The automated scans are taking in the latest definitions for each run.

You can report a false positive at https://www.microsoft.com/wdsi/filesubmission

**Pull request merged**